ckanext/datapusher/templates/package/resource_edit_base.html

```diff
@@ -0,0 +1,6 @@
+{% ckan_extends %}
+
+{% block inner_primary_nav %}
+  {{ super() }}
+  {{ h.build_nav_icon('resource_data', _('DataPusher'), id=pkg.name, resource_id=res.id) }}
```

Comment:
I just find it strange that the "DataStore" tab only appears when the datapusher plugin is enabled.. I can switch it back though